pubnub/event_engine/manage_effects.py

```diff
@@ -28,51 +30,56 @@ def __init__(self, pubnub_instance, event_engine_instance,
         self.event_engine = event_engine_instance
         self.pubnub = pubnub_instance
 
+        self.logger = logging.getLogger("pubnub")
```

Comment:
I have a question. 
Is that a common thing to just assigning logger to the field in pubnub? There is no better way to achieve same thing?